community/opendoas/files/doas.conf

```diff
@@ -0,0 +1,5 @@
+### Select one of the following. Deleting these may cause package installation
+### and removal to function unexpectedly
+
+permit setenv { KISS_FORCE } :wheel cmd kiss
+#permit nopass setenv { KISS_FORCE } :wheel cmd kiss
```

Comment:
No more `persist`?

Reply:
I've just learned that it's potentially dangerous on Linux as it requires a different implementation than OpenBSD

Reply:
Understood. :+1: